### ISSUE TITLE
Fresh install phpvirtualbox docker

### ISSUE THREAD

**Issue by DopeDealers** [jazzdd86/phpVirtualbox#21]
Hello! im writing this because im using your image with docker-compose and I cant seem to login to the web interface

`version: '3'
services:
    vbox_http:
        container_name: vbox_http
        restart: always
        ports:
            - "8085:80"
        environment:
            SRV1_HOSTPORT: "vbox_websrv_1:18083"
            SRV1_NAME: "Cyci: Miami"
            SRV1_USER: "ubuntu"
            SRV1_PW: "pass"
        depends_on:
            - vbox_websrv
        image: jazzdd/phpvirtualbox

    vbox_websrv:
        container_name: vbox_websrv_1
        restart: always
        volumes:
            - "./ssh:/root/.ssh"
        environment:
            USE_KEY: 1
            SSH_PORT: port
        image: jazzdd/vboxwebsrv
        command: ubuntu@ip`

removed sensitive data but I get this error
`Could not connect to host (http://vbox_websrv_1:18083/)`

any step in the right direction would be nice
along with that I use an SSH key with a passphrase idk if that has anything to do with it but thought id include it

**Comment by jazzdd86** [comment]
Hi, did you have a look in the vbox_websrv logs if the ssh connection works? If yes, is the vbox_websrv_1 service running out of the same docker compose file? If no, they need to be in the same virtual network.

**Comment by DopeDealers** [comment]
Yes I looked after I'm assuming it's the pass phrase like I mentioned. Any work around?

**Comment by jazzdd86** [comment]
can you try it without a passphrase?
